Source/CompanyCommunicator.Common/Adapter/CCBotAdapterBase.cs

```diff
@@ -8,16 +8,26 @@ namespace Microsoft.Teams.Apps.CompanyCommunicator.Common.Adapter
     using System.Threading;
     using System.Threading.Tasks;
     using Microsoft.Bot.Builder;
-    using Microsoft.Bot.Builder.Integration;
     using Microsoft.Bot.Builder.Integration.AspNet.Core;
     using Microsoft.Bot.Connector.Authentication;
     using Microsoft.Bot.Schema;
+    using Microsoft.Extensions.Logging;
 
     /// <summary>
     /// Bot Framework Http Adapter interface.
     /// </summary>
-    public interface ICCBotFrameworkHttpAdapter : IAdapterIntegration, IBotFrameworkHttpAdapter
+    public abstract class CCBotAdapterBase : CloudAdapter
```

Comment:
What's the purpose of interposing this abstract class in the hierarchy?

Reply:
In the current state, we had 2 classes that were implementing the interface. Now that we're extending CloudAdapter, switched to an abstract class.